src/Storefront/Resources/app/storefront/src/main.js

```diff
@@ -167,8 +169,6 @@ document.addEventListener('readystatechange', (event) => {
 /*
 run utils
 */
-new AjaxModalExtensionUtil();
```

Comment:
When you remove all usages of that class. Webpack will remove it entirely from the output, cause of the three shacking. So all plugins will break, which currently uses that.

Reply:
AFAIK the class is not published. As everyone who needs it compiles it into their source code using an import they are not shaken off.

<img width="394" alt="grafik" src="https://user-images.githubusercontent.com/1133593/89131820-d8fe6d80-d50f-11ea-9327-3afab428eb08.png">

Reply:
Webpack currently does not bundle already bundled things again in the plugins. So the plugins are using the class needs to rebuild the js file.
We had the same problem last week with StoreHttpClient. So we made it bound to window. https://github.com/shopware/platform/commit/0984308e85e2fa6dc8af706bb21e7eacb123a757

Reply:
Oh, ok. It is smart. I'll bind it to window as well :)

Reply:
@JoshuaBehrens any progress? :-)